packages/venia-concept/src/components/Checkout/submitButton.js

```diff
@@ -0,0 +1,26 @@
+import { Component, createElement } from 'react';
+import { func, string } from 'prop-types';
+
+import Button from 'src/components/Button';
+
+const isDisabled = status => ['ACCEPTED', 'SUBMITTING'].includes(status);
+
+class SubmitButton extends Component {
+    static propTypes = {
+        status: string,
+        submitOrder: func
```

Comment:
This one definitely should be required, since it has no purpose without an onClick callback.